linera-service/src/cli_wrappers/local_kubernetes_net.rs

```diff
@@ -80,6 +80,17 @@ impl LineraNetConfig for LocalKubernetesNetConfig {
     }
 }
 
+impl Drop for LocalKubernetesNet {
+    fn drop(&mut self) {
+        // Block the current runtime to cleanup
+        let handle = tokio::runtime::Handle::current();
+        let _guard = handle.enter();
+        futures::executor::block_on(async move {
```

Comment:
Please read the docs: https://docs.rs/tokio/latest/tokio/runtime/struct.Handle.html#method.block_on

Reply:
What I tried first was `Runtime.block_on`, but what happens is that if I try to just create a Runtime here, then I get this error: `Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.`
So a solution for this I thought was to spawn a thread, and start the `Runtime` from that thread, but then I get no guarantee that the cleanup will actually finish before the process gets killed. And indeed it was getting killed before all clusters were deleted.
So then I ended up with this solution. But TIL about the `block_on` on `Handle`! Let me try that :)

Reply:
I get the same `Cannot start a runtime from within a runtime. This happens because a function (like `block_on`) attempted to block the current thread while the thread is being used to drive asynchronous tasks.` error using `Handle.block_on` instead here :/